playbook/app/pb_kits/playbook/pb_dropdown/dropdown.html.erb

```diff
@@ -8,9 +8,13 @@
         <%= pb_rails("caption", props: {text: object.label, margin_bottom:"xs"}) %>
     <% end %>
     <div class="dropdown_wrapper<%= error_class %>" style="position: relative">
-        <input type="hidden" name="<%= object.name %>" id="dropdown-selected-option" value="<%= input_default_value %>" />
-        <input id="dropdown-form-validation" name="<%= object.name %>_form_validation" value="<%= input_default_value %>" style="display: none" <%= object.required ? "required" : ""%> />
-
+        <input 
+            data-default-value="<%= input_default_value %>" 
+            id="dropdown-selected-option" 
```

Comment:
should the dev be able to pass their own id with `input_options`?

Reply:
Interesting point! This id is being used to make the dropdown work, under the hood, if we wanted devs to be able to change that id, we'd have to change our implementation to maybe use a data attribute to hook into 🤔
We also have not added input_options to this kit yet, which may be a follow up story!